examples/scenes/src/download/default_downloads.rs

```diff
@@ -39,5 +39,65 @@ pub(super) fn default_downloads() -> Vec<SVGDownload> {
             url: "https://upload.wikimedia.org/wikipedia/commons/5/58/Coat_of_arms_of_the_Kingdom_of_Yugoslavia.svg".to_string(),
             name: "Coat of Arms of the Kingdom of Yugoslavia".to_string()
         },
+        SVGDownload {
+            builtin:Some(BuiltinSvgProps {
+                info: "resvg",
```

Comment:
I'd like this to be slightly more detailed - maybe a link to https://github.com/RazrFalcon/resvg-test-suite?